.github/workflows/podman-desktop-e2e-windows.yaml

```diff
@@ -0,0 +1,158 @@
+name: Podman Desktop E2E Happy Path test
+
+on:
+  workflow_dispatch:
+    inputs:
+      fork:
+        default: 'containers'
+        description: 'Podman Desktop repo fork'
+        type: string
+        required: true
+      branch:
+        default: 'main'
+        description: 'Podman Desktop repo branch'
+        type: string
+        required: true
+
+jobs:
+  windows:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        windows-version: ['10']
+        windows-featurepack: ['22h2-ent']
+
+    steps:
+    - name: Create instance
+      run: |
+        # Create instance
+        podman run -d --name windows-create --rm \
+          -v ${PWD}:/workspace:z \
+          -e ARM_TENANT_ID=${{ secrets.ARM_TENANT_ID }} \
+          -e ARM_SUBSCRIPTION_ID=${{ secrets.ARM_SUBSCRIPTION_ID }} \
+          -e ARM_CLIENT_ID=${{ secrets.ARM_CLIENT_ID }} \
+          -e ARM_CLIENT_SECRET='${{ secrets.ARM_CLIENT_SECRET }}' \
+          quay.io/rhqp/qenvs:v0.0.5 azure \
+            windows create \
+            --project-name 'windows-desktop' \
+            --backed-url 'file:///workspace' \
+            --conn-details-output '/workspace' \
+            --windows-version '${{ matrix.windows-version }}' \
+            --windows-featurepack '${{ matrix.windows-featurepack }}' \
+            --vmsize 'Standard_D8a_v4' \
+            --tags project=podman-desktop \
+            --spot
+        # Check logs 
+        podman logs -f windows-create
+
+    - name: Check instance system info
+      run: |
+        ssh -i id_rsa \
+          -o StrictHostKeyChecking=no \
+          -o UserKnownHostsFile=/dev/null \
+          -o ServerAliveInterval=30 \
+          -o ServerAliveCountMax=1200 \
+          $(cat username)@$(cat host) "systeminfo"
+
+    - name: Emulate X session 
+      run: |
+        # use fake rdp to emulate an active x session
+        podman run -d --name x-session \
+          -e RDP_HOST=$(cat host) \
+          -e RDP_USER=$(cat username) \
+          -e RDP_PASSWORD=$(cat userpassword) \
+          quay.io/rhqp/frdp:v0.0.1
+        # Wait until the x session has been created
+        podman wait --condition running x-session
+        # Check logs for the x session
+        podman logs x-session
+
+    - name: Run PDE2E builder and setup image
+      run: |
+        podman run -d --name pde2e-builder-run \
```

Comment:
From the script I see you build PD based on fork and branch so you would be able to build from PR, but should not be easier to just pass the PR-Number?

Also I see you install dependencies without choco so I guess I can close https://github.com/adrianriobo/qenvs/pull/143

Reply:
> From the [script](https://github.com/odockal/pde2e-builder/blob/5dd843ac31d732341289e3a3ae8a000a89783720/lib/windows/run.ps1#L85C34-L85C41) I see you build PD based on fork and branch so you would be able to build from PR, but should not be easier to just pass the PR-Number?

Not intended for PR check now. 

> 
> Also I see you install dependencies without choco so I guess I can close [adrianriobo/qenvs#143](https://github.com/adrianriobo/qenvs/pull/143)

Yes please.